integrationtest/core/src/it/scala/io/gearpump/integrationtest/checklist/StabilitySpec.scala

```diff
@@ -70,7 +70,6 @@ class StabilitySpec extends TestSpecBase {
   "kill worker" should {
     "worker will not recover but all its executors will be migrated to other workers" in {
       // setup
-      restartClusterRequired = true
```

Comment:
Without this line, the next test case will have one worker less. Consider all test cases are independent or executed randomly, what is the sequence, if cluster has one worker less?